bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -60,35 +60,59 @@ impl Default for Counter {
 }
 
 pub struct CertKeyPair {
-    cert_path: &'static str,
-    cert: &'static [u8],
-    key: &'static [u8],
+    cert_path: String,
+    key_path: String,
+    ca_path: String,
+    cert: Vec<u8>,
+    key: Vec<u8>,
 }
 
 impl Default for CertKeyPair {
     fn default() -> Self {
-        CertKeyPair {
-            cert_path: concat!(
-                env!("CARGO_MANIFEST_DIR"),
-                "/../../../tests/pems/rsa_4096_sha512_client_cert.pem",
-            ),
-            cert: &include_bytes!("../../../../tests/pems/rsa_4096_sha512_client_cert.pem")[..],
-            key: &include_bytes!("../../../../tests/pems/rsa_4096_sha512_client_key.pem")[..],
-        }
+        let prefix = concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/../../../tests/pems/rsa_4096_sha512_client_"
+        );
+        Self::from(prefix, "cert", "key", "cert")
     }
 }
 
 impl CertKeyPair {
-    pub fn cert_path(&self) -> &'static str {
-        self.cert_path
+    pub fn from(prefix: &str, chain: &str, key: &str, ca: &str) -> Self {
+        let cert_path = format!("{prefix}{chain}.pem");
+        let key_path = format!("{prefix}{key}.pem");
+        let ca_path = format!("{prefix}{ca}.pem");
+        let cert = std::fs::read(&cert_path)
+            .unwrap_or_else(|_| panic!("Failed to read cert at {cert_path}"));
+        let key =
+            std::fs::read(&key_path).unwrap_or_else(|_| panic!("Failed to read key at {key_path}"));
```

Comment:
```suggestion
        let cert = std::fs::read(&cert_path).expect("Failed to read cert at {cert_path}");
        let key = std::fs::read(&key_path).expect("Failed to read key at {key_path}");
```
I think we can just use `expect` here.

Reply:
That's what I had, but clippy flagged it. I think because the `{cert_path}` means that it's quietly doing `format!`, so it only wants to allocate the string if it needs to.